src/kaleidoscope/key_defs_sysctl.h

```diff
@@ -18,41 +18,50 @@
 
 // System control mappings
 
-#define System_PowerDown Key(HID_SYSTEM_POWER_DOWN, KEY_FLAGS | SYNTHETIC|IS_SYSCTL | HID_TYPE_OSC)
-#define System_Sleep Key(HID_SYSTEM_SLEEP, KEY_FLAGS | SYNTHETIC|IS_SYSCTL | HID_TYPE_OSC)
-#define System_WakeUp Key(HID_SYSTEM_WAKE_UP, KEY_FLAGS | SYNTHETIC|IS_SYSCTL | HID_TYPE_OSC)
-#define System_ContextMenu Key(HID_SYSTEM_CONTEXT_MENU, KEY_FLAGS | SYNTHETIC|IS_SYSCTL | HID_TYPE_OSC)
-#define System_MainMenu Key(HID_SYSTEM_MAIN_MENU, KEY_FLAGS | SYNTHETIC|IS_SYSCTL | HID_TYPE_OSC)
-#define System_AppMenu Key(HID_SYSTEM_APP_MENU, KEY_FLAGS | SYNTHETIC|IS_SYSCTL | HID_TYPE_OSC)
-#define System_MenuHelp Key(HID_SYSTEM_MENU_HELP, KEY_FLAGS | SYNTHETIC|IS_SYSCTL | HID_TYPE_OSC)
-#define System_MenuExit Key(HID_SYSTEM_MENU_EXIT, KEY_FLAGS | SYNTHETIC|IS_SYSCTL | HID_TYPE_OSC)
-#define System_MenuSelect Key(HID_SYSTEM_MENU_SELECT, KEY_FLAGS | SYNTHETIC|IS_SYSCTL | HID_TYPE_OSC)
-#define System_MenuRight Key(HID_SYSTEM_MENU_RIGHT, KEY_FLAGS | SYNTHETIC|IS_SYSCTL | HID_TYPE_RTC)
```

Comment:
Not all of the System control keys -are- OSC keys.  Even if we're not yet using this data, I really don't want to just discard it from the tables.

Reply:
The problem is that the `HID_TYPE_*` bits overlap with other bits that are in use in the `flags` byte of a `Key` object. If you include `HID_TYPE_OOC`, you're also including `IS_CONSUMER` along with it. `handleSyntheticKeyswitchEvent()` could be changed to check for System keys before Consumer keys to solve that problem, but Consumer keys can't use those `HID_TYPE_*` constants.

Reply:
I want @algernon's input before I sign off on it, but I think I'd be ok with a solution where, for now, we define all the HID_TYPE_XX constants to 0, rather than just removing them from the key definitions.

Reply:
Correction: we can't solve it by changing `handleSyntheticKeyswitchEvent()`, because Consumer keys might have the `IS_SYSCTL` bit set, depending on the value.

Reply:
I think we could call out four HID types (using the two bits above `IS_CONSUMER` and below `SYNTHETIC`). Or a default, and three that get special handling.

Reply:
I'd rather defer the exact bit math until we understand our needs.